torchmultimodal/modules/losses/contrastive_loss_with_temperature.py

```diff
@@ -158,6 +158,10 @@ class ContrastiveLossWithTemperature(nn.Module):
         logit_scale (Union[float, nn.Module]): Log of the learnable temperature parameter value
             A nn.Parameter instantiation can also be passed directly in case parent class
             is handling the initialization.
+        logit_scale_min (Optional[float]): Log of the minimum temperature value.
+            If None, then temperature will not be clamped to a minimum value.
```

Comment:
Is this true? I thought it will be clamped to the default value

Reply:
From [pytorch docs](https://pytorch.org/docs/stable/generated/torch.clamp.html#:~:text=If%20min%20is%20None%2C%20there%20is%20no%20lower%20bound.%20Or%2C%20if%20max%20is%20None%20there%20is%20no%20upper%20bound.): "If min is None, there is no lower bound. Or, if max is None there is no upper bound."

Reply:
I see, I was referring more to the default behavior here. In other words, if the user does not pass logit_scale_min or logit_scale_max we will clamp to the default values from CLIP. Would be worth mentioning this in the docstring as well.

Btw might want to do a check that at least one of logit_scale_min and logit_scale_max is not None, otherwise clamp will throw an error.

Reply:
oh ok, I added defaults for all args in the docstring and a ValueError for the both None case